### PR TITLE
Add qtile-top to measure memory usage

### PR DESCRIPTION
**Pull request by Roger**
4arjy

### CODE REVIEW

**Comment by ramnes** [comment]
Awesome! :dancer:

File: bin/qtile-top [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+import curses
+
+import linecache
+import tracemalloc
+from tracemalloc import Snapshot
+from libqtile import command
+
+
+class TraceNotStarter(Exception):
```
Comment (pankshok) [inline bin/qtile-top]: Hmmm, maybe TraceNotStarted? I just suppose.

File: setup.py [hunk 2]
```diff
@@ -140,6 +140,7 @@ def finalize_options(self):
         "bin/qsh",
         "bin/qtile",
         "bin/qtile-run",
+        "bin/qtile-top",
```
Comment (flacjacket) [inline setup.py]: Oh, you already added this to setup.py, ignore my comment about that on IRC

Comment (flacjacket) [inline setup.py]: Oh, nevermind, I see the commits now, cool

**Pull request closed without merging**